frontend/src/components/homepage/PreviewModal.tsx

```diff
@@ -86,36 +98,42 @@ const PreviewModal = ({
       isOpen={preview}
       onClose={previewBook}
       motionPreset="slideInBottom"
-      size="4xl"
+      size="6xl"
     >
       <ModalOverlay />
-      <ModalContent>
+      <ModalContent paddingLeft="20px">
         <ModalCloseButton position="static" marginLeft="auto" />
         <ModalHeader paddingTop="-10px">
-          <Heading as="h3" size="md">
+          <Heading as="h3" size="lg" marginBottom="10px" width="75%">
             {title}
           </Heading>
-          <Box padding="10px 0px 10px 0px" float="left">
-            <Badge>{`Level ${level}`}</Badge>
-            <Badge>{`${convertLanguageTitleCase(language)}`}</Badge>
-          </Box>
+          <Badge variant="language" size="s">{`${convertLanguageTitleCase(
+            language,
+          )}`}</Badge>
+          <Badge
+            variant={getLevelVariant(level)}
+            size="s"
+          >{`Level ${level}`}</Badge>
+          <Badge variant="stage" size="s">
+            {storyTranslationId ? "Review" : "Translation"}
+          </Badge>
           <Button
             float="right"
-            width="21%"
+            width="20%"
             colorScheme="blue"
-            mr={3}
+            size="secondary"
+            marginRight="15px"
             onClick={primaryBtnOnClick()}
           >
             {primaryBtnText}
```

Comment:
this text seems to be returning  `view translation` or `edit translation` rather than `translate` or `review` (i think because it is passed in as a prop from the StoryCard, and the StoryCard `primaryBtnText()` returns from the `isMyStory=true` branch)
https://github.com/uwblueprint/planet-read/blob/27e64b571cd4571e66e87b3b8f79c139da772ec6/frontend/src/components/homepage/StoryCard.tsx#L122-L124